etc/config/c.amazon.properties

```diff
@@ -179,9 +179,21 @@ group.ccross.groupName=Cross GCC
 
 ###############################
 # GCC for PPC
-group.cppc.compilers=cppcg48:cppc64leg630
-group.cppc.groupName=PowerPC GCC
+group.cppc.compilers=cppcg48:cppc64leg630:ppc64leg8:ppc64g8:ppc64clang:ppc64leclang
```

Comment:
This IDs are going to need to change for this to be mergable. Could you please add a `c` before the ids (As done for `cppcg48` for example). Thanks! :)

Reply:
done. sorry I missed that.